core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalLogWriter.java

```diff
@@ -364,6 +320,35 @@ private void createNewLogFile() throws IOException {
     LOG.info("Created current log file: {}", currentLog);
   }
 
+  private void completeLog(UfsJournalFile currentLog, long nextSequenceNumber) throws IOException {
```

Comment:
Could you add javadoc for this, explaining what this method will do and what the parameters should be?

Reply:
done